ext/natives/codegen_cfx_natives.lua

```diff
@@ -745,12 +745,12 @@ native 'GET_VEHICLE_WHEEL_SPEED'
 	returns 'float'
 	doc [[
 	<summary>
-	Gets speed of a wheel at the tyre. 
+	Gets speed of a wheel at the tyre.
 	Max number of wheels can be retrieved with the native GET_VEHICLE_NUMBER_OF_WHEELS.
 	</summary>
 	<returns>An integer.</returns>
 ]]
-	
```

Comment:
Useful change! Have people still not learned to check diffs to see if their editor has decided to be 'smart' in a third-party codebase?

Please fix and **force-push** to the PR branch. Making a new commit in the same branch, contrary to popular belief, will not fix this kind of screwups.